iot-hub/Samples/service/JobsSample/JobsSample.cs

```diff
@@ -40,50 +41,58 @@ public async Task RunSampleAsync()
 
 
             // *************************************** Schedule twin job ***************************************
-            Console.WriteLine($"Schedule twin job {jobId} for {DeviceId}...");
-            JobResponse createJobResponse = await _jobClient
-                .ScheduleTwinUpdateAsync(
-                    jobId,
-                    query,
-                    twin,
-                    DateTime.UtcNow,
-                    (long)TimeSpan.FromMinutes(2).TotalSeconds);
-
-            Console.WriteLine("Schedule response");

```

Comment:
There is no need for this line and below to be in the try/catch block since they won't throw a ThrottlingException.

Try something like

```csharp

JobResponse createJobResponse;

try
{
            createJobResponse = await _jobClient
                .ScheduleTwinUpdateAsync(
                    jobId,
                    query,
                    twin,
                    DateTime.UtcNow,
                    (long)TimeSpan.FromMinutes(2).TotalSeconds);
}
catch (ThrottlingException)
{
    Console.WriteLine("...");
    return;
}

Console.WriteLine("Schedule response");
```

Reply:
awesome, I'll get on changing this and moving it to other places where it's needed. Thanks Tim!